jac-uk/admin
# Custom report improvements

## What's included?
Logic to display location prefs, part time working prefs and additional working prefs questions in custom report drop down.
Ties in with digital-platform [#740](https://github.com/jac-uk/digital-platform/pull/740).

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Find exercise with additional working preferences.
Navigate to custom report, observe questions are displayed in drop down and on report when it renders.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context
![image](https://user-images.githubusercontent.com/44227249/186944036-723bcb0f-c624-4571-b0bd-c291992870f7.png)

PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
